client/src/modules/stock/settings/stock-settings.html

```diff
@@ -0,0 +1,78 @@
+<div class="flex-header">
+  <div class="bhima-title">
+    <ol class="headercrumb">
+      <li class="static" translate>TREE.STOCK</li>
+      <li class="title" translate>TREE.STOCK_SETTINGS</li>
+    </ol>
+    <div class="flex-content">
```

Comment:
You need to close the `<div class="flex-header">` before you can put the flex content.  You'll get very interesting results otherwise!